rust/libnewsboat/src/utils.rs

```diff
@@ -265,8 +267,78 @@ pub fn unescape_url(rs_str: String) -> Option<String> {
     Some(result.unwrap().replace("\0",""))
 }
 
+pub fn get_command_output(cmd: &str) -> String {
+    let cmd = Command::new("sh").arg("-c").arg(cmd).output();
```

Comment:
Why does `get_command_output` need to spawn a shell (especially compared to `run_command`)? I realize this is the same behavior as before, but maybe it would be a good time to actually document this behavior.

Reply:
Because it needs the shell to interpret the arguments. `run_command` only takes one argument, so there is no need to tokenize.

We could implement the POSIX spec ourselves, but it won't gain us anything.

I'll add a comment about the shell, and postpone proper documenting until after #334 is done.

Reply:
> We could implement the POSIX spec ourselves, but it won't gain us anything.

That would indeed be pointless.